### PR TITLE
add readme validate dependencies

### PR DESCRIPTION
**Pull request by GalRabin**
## Status
In Progress

## Description
Adding the following for docker build image:
1. `package.json` with dependencies for validating readme.
2. setting the variable to true.

### CODE REVIEW

File: docker/content-build/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "description": "Readme validation dependecies",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "dependencies": {
+    "@mdx-js/mdx": "^1.5.7",
+    "commander": "^5.0.0",
+    "fs-extra": "^8.1.0",
+    "jsdoc-to-markdown": "^5.0.2"
```
Comment (bakatzir) [inline docker/content-build/package.json]: why is this needed?

**Comment by bakatzir** [comment]
@orhovy FYI

File: docker/content-build/package.json (hunk 1 above)
Comment (bakatzir) [reply docker/content-build/package.json]: was beforehand.

**Comment by bakatzir** [comment]
We need to create a separated docker for the demsito-sdk.

**Comment by glicht** [comment]
@bakatzir @GalRabin I think it will be better to manage this in .circleci/config.yml as we do for the python deps. We reached the conclusion that doing it in docker introduces too much overhead.

**Comment by bakatzir** [comment]
fixes: https://github.com/demisto/etc/issues/21123

**Comment by GalRabin** [comment]
> @bakatzir @GalRabin I think it will be better to manage this in .circleci/config.yml as we do for the python deps. We reached the conclusion that doing it in docker introduces too much overhead.

https://github.com/demisto/demisto-sdk/pull/262 - @glicht  Done as you recommended
